eng/pipelines/templates/stages/archetype-js-release.yml

```diff
@@ -236,6 +236,11 @@ stages:
           - ${{if ne(artifact.skipPublishDevFeed, 'true')}}:
             - pwsh: |
                 $detectedPackageName=Get-ChildItem $(Pipeline.Workspace)/${{parameters.ArtifactName}}/${{artifact.name}}/*.tgz
+                if ($detectedPackageName -notmatch "-alpha")
```

Comment:
Lets add this check in the build stage as well if SetDevVersion is set to true.

Reply:
not sure if this will happen but if the package name ends up with "-alpha" in it, this will no catch that case but I suspect that is not likely.

Reply:
We do not build alpha version of private packages so adding this check at build time will fail for them without more checks to skip them as well.